Running 5 test cases...
big_int_test.cpp(386): error in "io_test": check iss1.good() == iss2.good() failed [true != false]
big_int_test.cpp(387): error in "io_test": check iss1.eof() == iss2.eof() failed [false != true]
big_int_test.cpp(388): error in "io_test": check iss1.fail() == iss2.fail() failed [false != true]
big_int_test.cpp(388): error in "io_test": check iss1.fail() == iss2.fail() failed [false != true]
big_int_test.cpp(386): error in "io_test": check iss1.good() == iss2.good() failed [true != false]
big_int_test.cpp(388): error in "io_test": check iss1.fail() == iss2.fail() failed [false != true]
big_int_test.cpp(386): error in "io_test": check iss1.good() == iss2.good() failed [true != false]
big_int_test.cpp(388): error in "io_test": check iss1.fail() == iss2.fail() failed [false != true]
big_int_test.cpp(386): error in "io_test": check iss1.good() == iss2.good() failed [true != false]
big_int_test.cpp(388): error in "io_test": check iss1.fail() == iss2.fail() failed [false != true]
big_int_test.cpp(429): error in "io_test": check test_number == test_number_copy failed [0 != 87612319782231237389123382]
